materialized_sql_view/model/abstract_materialized_sql_view.py

```diff
@@ -0,0 +1,344 @@
+# -*- coding: utf-8 -*-
+
+import psycopg2
+import logging
+from openerp.osv import osv
+from openerp import SUPERUSER_ID
+from abc import ABCMeta, abstractmethod
+
+logger = logging.getLogger(__name__)
+ABSTRACT_MODEL_NAME = 'abstract.materialized.sql.view'
+
+
+class AbstractMaterializedSqlView(osv.AbstractModel):
+
+    """This class is an abstract model to help developer to
+       create/refresh/update materialized view.
+    """
+    _name = ABSTRACT_MODEL_NAME
+    _description = u"This is an helper class to manage materialized SQL view"
+    _auto = False
+
+    _sql_mat_view_name = None
+    """The name of the materialized sql view.
+       Must be defined in inherit class (using inherit = [])
+    """
+
+    @property
+    def sql_mat_view_name(self):
+        if not self._sql_mat_view_name:
+            self._sql_mat_view_name = self._table
+        return self._sql_mat_view_name
```

Comment:
Can you add a docstring here to explain that self._sql_mat_view_name is expected to be changed manually?